FairyMAGs Hackathon 2025-10-06 (#2977)
galaxyproject/galaxy-hub: content/events/2025-10-06-FairyMAGs-hackathon/index.md

date: '2025-10-06'
end: '2025-10-09'
location:
  name: "Freiburg, Germany"
contact: "Paul Zierep, Bérénice Batut"
tags: [Hackathon]
supporters:
  - ELIXIR
subsites: [eu]
---

# FairyMAGs Hackathon: Optimising Metagenomics Assembled Genomes building 🚀 

MAGs workflows are both **technically and scientifically challenging**, yet they are **essential tools** for reducing microbial dark matter, as most microbes cannot be cultivated.  

As part of the [FairyMAGs project](https://artifact.galaxyproject.org/news/2025-01-09-bfsp-fairy-ma-gs/) withhin the [ELIXIR BFSP Programme](https://elixir-europe.org/internal-projects/commissioned-services/science/biodiversity-security-pathogens), we are organizing a **4-day hackathon** and warmly invite the community to join us in Freiburg. This event is an opportunity to **collaborate, discuss, and improve MAGs workflows together!**  

FAIRyMAGs is led by Paul Zierep and Bérénice Batut in collaboration with four ELIXIR nodes, that will be participating the hackathon.

* ELIXIR France (Bérénice Batut)
* ELIXIR Germany (Paul Zierep)
* ELIXIR Italy (Giuseppe Defazio and Bruno Fosso)
* EMBL-EBI (Martin Beracochea, Santiago Sanchez)

> # 🔬 What?
> ## A 4-day scientific and technical hackathon focused on:

Reviewer: ```suggestion
# Objectives

The objectives of this 4-day scientific and technical hackathon are to optimize Metagenomics Assembled Genomes building by:
```

### 🚀 Enhancing FAIR MAGs Workflows

> * 🧩 Optimizing assembly strategies
>   * Individual assembly vs. co-assembly vs. grouped-assembly
>   * Automating decision-making for assembly selection
> * 🔍 Bin refinement improvements
>   * Where do we go beyond MetaWrap?
>   * 🏗️ Should we integrate more binning tools for better results?

Reviewer: ```suggestion
  - Optimizing assembly strategies: Individual assembly vs. co-assembly vs. grouped-assembly, Automating decision-making for assembly selection
  - Bin refinement improvements: Where do we go beyond MetaWrap? Should we integrate more binning tools for better results?
```

> ### 📚 Developing user-friendly training materials
> * 🛠️ What parameters should users adjust?
> * 🧑‍🏫 How to make complex concepts more accessible?

Reviewer: ```suggestion
- 📚 **Developing user-friendly training materials**
```

> ### 📊 Advancing evaluation methods (using CAMI infrastructure & real data)

Reviewer: ```suggestion
- 📊 **Advancing workflow evaluation methods** (using CAMI infrastructure & real data)
```

> * 🧬 Have metagenomic WGS samples? Want to create MAGs? This is your chance!

Reviewer: ```suggestion

       Have metagenomic WGS samples? Want to create MAGs? This is your chance!
```

### ⚙️ Building intelligent resource estimation tools

> * 🚦 Whether you're using Nextflow, Snakemake, or Galaxy, resource allocation is key
> * ❌ Over-provisioning wastes resources; under-provisioning leads to failed jobs
> * 🌍 Let’s design workflow-agnostic tools that optimize resource usage, reducing computational waste and environmental impact!

Reviewer: ```suggestion

    Whether you're using Nextflow, Snakemake, or Galaxy, resource allocation is key. Over-provisioning wastes resources; under-provisioning leads to failed jobs. Let’s **design** **workflow-agnostic tools** that **optimize resource usage**, **reducing computational waste and environmental impact**!
```

---

> title: "FairyMAGs Hackathon 2025: Optimising Metagenomics Assembled Genomes building 🚀"

Reviewer: ```suggestion
title: "FAIRyMAGs Hybrid Hackathon 2025: Optimising Metagenomics Assembled Genomes building 🚀"
```

date: '2025-10-06'
end: '2025-10-09'
location:
  name: "Freiburg, Germany"

> contact: "Paul Zierep, Bérénice Batut"

Reviewer: ```suggestion
continent: EU
contact: "Paul Zierep, Bérénice Batut"
```

> tags: [Hackathon]

Reviewer: ```suggestion
tags: [hackathon, microbial, metagenomics, assembly, training]
```

supporters:

>   - ELIXIR

Reviewer: ```suggestion
  - ELIXIR
  - de.NBI
  - IFB
```

> subsites: [eu]

Reviewer: ```suggestion
subsites: [all]
```

> ---

Reviewer: ```suggestion
tease: "This 4-day hybrid hackathon aims to optimize Metagenomics Assembled Genomes Building workflows"
hide_tease: true
---
```

# Join Us for Optimising Metagenomics Assembled Genomes Building Workflows

Metagenomics Assembled Genomes building workflows are both **technically and scientifically challenging**, yet they are **essential tools** for reducing microbial dark matter, as most microbes cannot be cultivated.  

> As part of the [FAIRyMAGs project](https://artifact.galaxyproject.org/news/2025-01-09-bfsp-fairy-ma-gs/) within the [ELIXIR BFSP Programme](https://elixir-europe.org/internal-projects/commissioned-services/science/biodiversity-security-pathogens), we are organizing a **4-day hackathon (October 6-9th)** and 1-day for an optional social event (October 10th). We warmly invite the community to join us in Freiburg. This event is an opportunity to **collaborate, discuss, and improve MAGs workflows together!**  

Reviewer: ```suggestion
As part of the [FAIRyMAGs project](https://artifact.galaxyproject.org/news/2025-01-09-bfsp-fairy-ma-gs/) within the [ELIXIR BFSP Programme](https://elixir-europe.org/internal-projects/commissioned-services/science/biodiversity-security-pathogens), we are organizing a **4-day hackathon (October 6-9th)** and 1-day for an optional social event (October 10th). We warmly invite the community to join us either **in person in Freiburg** or **online**. This event is an opportunity to **collaborate, discuss, and improve MAGs workflows together!**  
```

| Time | Topic |
|---|---|
| 10:00 | Welcome & Logistics |
| 10:10 | Presentation of personal goals and plans for the week |
| 11:00 | Presentation of the FairyMAGs project |
| 11:30 | Focus Groups selection |
| 12:00 | Lunch |
| 13:00 | Hacking |

## Tuesday, October 7th

| Time | Topic |
|---|---|
| 9:30 | Hacking days - Stand-up meeting |
| 12:00 | Lunch |

## Wednesday, October 8th

| Time | Topic |
|---|---|
| 9:30 | Hacking days - Stand-up meeting |
| 12:00 | Lunch |

## Thursday, October 9th

| Time | Topic |
|---|---|
| 9:30 | Hacking |
| 12:00 | Lunch |
| 12:30 | Presentations of Hackathon outcomes |

## Friday, October 10th

Social event optional. Sight seeing in Freiburg or visit to [Mundenhof Tier Park](https://www.mundenhof.de/).

# Event rules and regulations

## Code of Conduct

The Hackathon will abide the [Galaxy Project CoC](https://galaxyproject.org/community/coc/).
Please, make sure that you read it before the event.

## Data protection policy

This event follows the data protection policy established by the University of Freiburg as the host of the event. For details please refer to https://uni-freiburg.de/en/data-protection-applications/

## Supporters

> <figure class="figure">
>   <img src="https://www.denbi.de/templates/nbimaster/img/denbi-logo-color.svg" alt="de.NBI Logo" width="150" style="margin-right: 30px;" />
>   <img src="https://elixir-europe.org/sites/default/files/white-orange-logo.png" alt="ELIXIR Logo" width="150" style="margin-right: 30px;" />
>   <img src="https://www.france-bioinformatique.fr/wp-content/uploads/logo-ifb-couleur.svg" alt="IFB Logo" width="150" />
> </figure>

Reviewer: ```suggestion
<div style="max-width: 300px">

![logo de.NBI](/images/logos/deNBILogo.png)
![logo IFB](/images/logos/IFB.png)
![logo ELIXIR](/images/logos/elixir-logo-big.png)

</div>
```
